api/v1alpha1/validation.go

```diff
@@ -681,3 +754,39 @@ func validateGraceDuration(schedule cron.Schedule, duration string) error {
 
 	return nil
 }
+
+// To ensure valid go-template syntax, we create a fake device that contains only ObjectMeta
+// (what we allow to include in parameters), then evaluate.
+type whitelistDevice struct {
+	Metadata ObjectMeta
+}
+
+func ValidateParametersInString(s *string, path string) (bool, []error) {
```

Comment:
Should we - and is there a way to - exclude some of go-template's actions like "if" ,"range", and "with" to protect the service?

Reply:
Do you mean to limit computational resources used?  I haven't seen a straight-forward way to do so.

Reply:
Please let us know if there will be limited syntax, or if we'll allow the broader syntax from go-templates. This may affect the UX.

Reply:
Yeah. Like sending an infinite / long-running loop.